modules/test/context_tests.go

```diff
@@ -51,6 +52,13 @@ func LoadUser(t *testing.T, ctx *context.Context, userID int64) {
 	ctx.User = models.AssertExistsAndLoadBean(t, &models.User{ID: userID}).(*models.User)
 }
 
+func LoadGitRepo(t *testing.T, ctx *context.Context) {
```

Comment:
Comment or CI failed.

Reply:
just fixed it :smile:

Reply:
No. you didn't, it should `LoadGitRepo` not `LoadRepo`.

Reply:
good catch, actually fixed now